### DISCUSSION TITLE
Add an option to Control Center to change sample rate

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#4639] [Features Request & Ideas | 特性请求 & 头脑风暴]
For some audio sensitive environments such as music production and audiophiles I believe there should be an option in the settings to change Bit Depth and Sample rate of your output/input devices. There may be as well a tool tip on how high sample rate may impact performance on low spec hardware.

**Comment by raspin0** [comment]
This is how it's done on windows 
![obraz](https://github.com/linuxdeepin/developer-center/assets/39202451/393bd38a-8501-407f-ab4e-9d5a85b5e61b)

**Comment by BLumia** [comment]
It seems currently no other DE supports this, but it might be a good idea.

cc @meiyixiang @Decodetalkers

**Reply by raspin0** [reply]
I believe it may not be present on other DEs, because values above 16bit-48khz are deemed to be an overkill for vast majority of people.
You need professional equipment, high quality audio and sensitive ears to hear any difference. 
Maybe it's not worth to put this in after all? My thought process was that there are people who have such equipment and would want to adjust these settings even if it's placebo.

**Comment by meiyixiang** [comment]
This is a bit of a question that is beyond my realm, and I need to understand it first and then see how to do it.
The sample rate is the number of times the sound waveform is sampled in 1 second, the higher the sample rate, the clearer and more specific the recording of the sound shape. For example, the standard sampling rate of VCD is 44100HZ, DVD is 48000HZ, and 96000/192000 is HD Blu-ray.
Bit depth is the dynamic range of recorded loudness, and the range of volume from maximum to minimum is dynamic range. The greater the bit depth, the more obvious the volume contrast of the sound and the richer the layers. 16Bit can provide a dynamic range of 96db (excluding high-frequency jitter, which is actually 92db), and the loudness range increases by about 6db for each additional bit of quantization accuracy. 24Bit provides about 144db of dynamic range. The sample rate determines the morphology of the sound (precision), and the bit rate determines the loudness range of the sound (dynamic range).
